strings/names/wizardsecond.txt

```diff
@@ -31,10 +41,59 @@ the Sorcelator
 the Spiral King
 the Unending
 the Unstoppable
+the Waxmancer
 the Weeping
 the White
 the Wise
 the Wizzard
-Unseen
-Weatherwax
-Yagg
\ No newline at end of file
+Infinitum
+Nauseam
+Borealis
+Fide
+Volente
+Sequitur
+Rasa
+the Annoying
+the Unseen
+the Obscure
+the Annoyed
+the Empty
+Voidfall
+Starfall
+Azurepelt
+Doomancer
+the Necromancer
+Midnight
+Who-was-blood
+Who-was-stone
+Who-was-light
+the Dappled
+Mountain-Mother
+the Moldywarp
+Evilton
+Among-many
+Deathbringer
+Lifereaper
+Shorekeeper
+Namekeeper
+Truenamer
+Evergreen
```

Comment:
Could you order this alphabetically? There's phyton sort scripts and tools to automatically do that for you.